addons/EmpyPopTracker/EmpyPopTracker.lua

```diff
@@ -224,7 +224,12 @@ end
 EmpyPopTracker.generate_info = function(nm, key_items, items)
     return {
         has_all_pops = not nm.pops or T(nm.pops):all(function(item)
-            return item.type == 'item' and owns_item(item.id, items) or owns_key_item(item.id, key_items)
+            -- Avoids confusing item and key item IDs
```

Comment:
Am I missing something here? Or is this change meant to be cosmetic only? Because it should do the same thing as before.

Reply:
That's what I thought. There's an issue wherein, if the `item.type` is "item" but the `owns_item` returns `false`, it falls back to running the `owns_key_items`. This can give us a false positive when we have a matching key item ID.

Reply:
Ah, that's a good point. Thanks for clarifying.